cookbooks/omnitruck/metadata.rb

```diff
@@ -4,11 +4,11 @@
 license 'Apache2'
 description 'Installs/Configures omnitruck'
 long_description 'Installs/Configures omnitruck'
-version '0.4.22'
+version '0.4.23'
 
 depends 'delivery-sugar'
 depends 'cia_infra'
-depends 'habitat', '~> 0.4.0'
+depends 'habitat', '~> 0.38.0'
```

Comment:
Do we even need to pin now?

Reply:
I had the same thought right after I ran `delivery review` :) If this works I'll add that as part of my cleanup